src/modules/dashboard/components/Extensions/ExtensionSetup.css

```diff
@@ -22,3 +22,15 @@
 .descriptionExample {
   font-style: italic;
 }
+
+.textArea {
+  margin-top: 43px;
+}
```

Comment:
Please make sure to make the textarea resizable verically _(currently you can't)_

How it's looking now:
![Screenshot from 2021-07-08 13-55-32](https://user-images.githubusercontent.com/1193222/124910441-448c6d00-dff4-11eb-9c5a-0a1d7d08ae4b.png)

What I want _(it shows the little thing in the corner that you can drag with your mouse)_:
![Screenshot from 2021-07-08 13-55-19](https://user-images.githubusercontent.com/1193222/124910494-51a95c00-dff4-11eb-87f5-a5675f53a7bf.png)

You can achieve this by setting:
```css
resize: vertical;
```